mkdocs.yml

```diff
@@ -85,19 +85,18 @@ extra:
       link: https://prairielearn.slack.com
 
 nav:
-  - Main: 'index.md'
+  - Overview: 'index.md'
   - Getting Started:
       - 'Requesting your course space': 'requestCourse.md'
-      - 'Creating content in the browser': 'getStarted.md'
+      - 'Concepts': 'concepts.md'
+      - 'Quick start': 'getStarted.md'
       - 'Workshop': 'workshop/index.md'
-  - Student Guide:
-      - 'Accessibility': 'student-guide/accessibility.md'
```

Comment:
Please don't move this. This student-specific page is an important part of student-facing PL documentation at many institutions and was added specifically to comply with accessibility requirements and pass accessibility reviews.